### PR TITLE
Feature/save poster

### PR DESCRIPTION
**Pull request by delilahrois**
PULL REQUEST:

Thank you for pushing your code!

What (if any) features are you implementing?
- Added function to save newly created poster to saved posters array. 

What (if anything) did you refactor?
- Refactored showMyPoster function to properly create Poster class instance.

Were there any issues that arose?
- No issues but function showSavedPosters is incomplete at this time.

Is there anything that you need from your teammate?
- Review and thoughts?

Any other comments, questions, or concerns?

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -184,6 +173,9 @@ function makePosterForm() {
 function showSavedPosters() {
   mainPoster.classList.add('hidden');
   savedPostersPg.classList.remove('hidden');
+  savedPostersGrid.innerHTML {
```
Comment (katiescruggs) [inline src/main.js:176]: If you're trying to change the savedPostersGrid here, the syntax is `savedPostersGrid.innerHTML = <newHTML>`. You can also use `appendChild` if you want to add things to the end of this HTML element: https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild.

Line 176 is currently causing an error due to these extra curly braces.

File: src/main.js [hunk 2]
```diff
@@ -212,10 +207,14 @@ function showMyPoster() {
 //returnToMain()
 //mainPage should display new saved poster
 
-function saveCreatedPoster() {
-  console.log("are you here")
-  var newPoster = new Poster({imageURL: posterImgURLInput, title: posterTitleInput, quote: posterQuoteInput});
-  images.push(posterImgURLInput.value);
-  titles.push(posterTitleInput.value);
-  quotes.push(posterQuoteInput.value);
+var currentPoster;
```
Comment (katiescruggs) [inline src/main.js:210]: Just convention, but usually global variables are all set at the top of the file.

**Review by katiescruggs — CHANGES_REQUESTED** [review]
Looks like something broke on this branch:
Screen Shot 2021-09-04 at 11 39 39 PM

When I pull down the `main` branch, I see that showing random posters is working, but on `feature/save-poster`, I get this error on line 176. This reminded me that I forgot to send you info on how to make the curly braces better colors in VS Code. There is an extension called `Bracket Pair Colorizer` if you want to check it out when you have time! 

I would recommend fixing this error on line 176 before merging this PR into main. Let me know if you want to look at anything together!

**Comment by susannaopal** [comment]
Thanks, Katie. We will be fixing this before submitting the final. And refactoring code to the best of our ability.

**Pull request merged**